geocodingv7/client.go

```diff
@@ -0,0 +1,147 @@
+package geocodingv7
```

Comment:
What do you think about calling it geocodingsearchv7? That seemed to be what the API was called, no?
Thinking if someone wants to add a Search endpoint later it might feel more correct to have the same package name?